vscode/src/debugger/session.ts

```diff
@@ -5,6 +5,8 @@
 
 import * as vscode from "vscode";
 
+import * as wasm from "../../../npm/lib/web/qsc_wasm.js";
```

Comment:
The npm package already exposes methods to load the wasm and call methods on it, which is what is used everywhere else (VS Code and browser). We shouldn't be loading files directly from the npm folder structure, but by importing from 'qsharp-lang' (and adding new methods and types there if needed)